webpack.commons.js

```diff
@@ -78,15 +78,7 @@ module.exports = {
           to: 'env'
         },
         {
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'white-logo.svg'),
-          to: 'images'
-        },
-        {
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'user.png'),
-          to: 'images'
-        },
-        {
-          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images', 'logo-tiny.png'),
+          from: path.resolve(__dirname, 'src', 'linagora.esn.unifiedinbox', 'images'),
```

Comment:
does this work? last time i checked I didn't copy the images folder for me

Reply:
Yes It works, for me in dev mode after `npm run serve` I can see all the images copied !

Reply:
alright, try compiling it in prod mode `npm run build:prod` and check the dist folder for the images. if everything is good we merge

Reply:
> does this work? last time i checked I didn't copy the images folder for me

FYI; in dev mode you can add `writeToDisk` prop in your dev server (webpack.common.js) and images are copied to `/dist/images`

Reply:
> alright, try compiling it in prod mode `npm run build:prod` and check the dist folder for the images. if everything is good we merge

Works, in prod mode, files are generated in `dist/assets/images`